projects/igniteui-angular/src/lib/accordion/accordion.component.ts

```diff
@@ -265,6 +322,11 @@ export class IgxAccordionComponent implements AfterContentInit, AfterViewInit, O
             panel.contentCollapsing.pipe(takeUntil(this.unsubChildren$)).subscribe((args: IExpansionPanelCancelableEventArgs) => {
                 this.panelCollapsing.emit(args);
             });
+            fromEvent(panel.header.elementRef.nativeElement, 'keydown')
```

Comment:
If you expose a getter for the focused element in the panel header (see previous comment), I think it could be used here as well (would look a bit better)